RefrigeratorExe/RefrigeratorExe/Shelf.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorExe
+{
+    internal class Shelf :IComparable<Shelf>
+    {
+        private int _floor;
+        private int _space;
+        public int ShelfSpace = 20;
+        public Guid Id{ get; }
+        public int Floor {
+            get
+            {
+                return _floor;
+            }
+            private set
+            {
+                if (value >= 0)
+                    _floor = value;
+                else
+                    throw new ArithmeticException("Illegal floor ,floor must be a positive number");
+            }
+        }
+        public int Space {
+            get
+            {
+                return _space;
+            }
+            private set
+            {
+                if (value >= 0)
+                    _space = value;
+                else
+                    throw new ArithmeticException("Illegal space , space must be a positive number");
+            }
+        }
+        public List<Item> Items;
+        public Shelf(int floor) 
+        {
+            try
+            {
+                Id = Guid.NewGuid();
+                Space = ShelfSpace;
+                Floor = floor;
+                Items = new List<Item>();
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message); 
+            }
+           
+        }
+        public override string ToString()
```

Comment:
consider this clean code:

public override string ToString()
{
    const string separator = "\n______________________________________________\n";
    var itemsList = string.Join(separator, Items);

    var result = $"Shelf Id: {Id}" +
                 $"\nFloor: {Floor}" +
                 $"\nSpace left: {Space}";

    if (!string.IsNullOrEmpty(itemsList))
        result += $"\n Items on the shelf:{separator}{itemsList}";
    else
        result += "\nNo items on this shelf";

    return result;
}